src/RP.ts

```diff
@@ -93,6 +94,11 @@ export class RP {
       return this;
     }
 
+    addCredentialFormats(credentialFormat: CredentialType): RP.Builder {
```

Comment:
Singular, as it only adds a single credential format to the array

Reply:
Fixed!